visualize.Rmd

```diff
@@ -1,115 +1,113 @@
+Puedes poner a prueba tu respuesta empleando el conjunto de datos `millas` que se encuentra en el paquete **datos** (`datos::millas`). Un conjunto de datos es una colección rectangular de variables (en las columnas) y observaciones (en las filas). `millas` contiene observaciones para 38 modelos de automóviles recopiladas por la Agencia de Protección Ambiental de los EE. UU. 
 
 ```{r}
-mpg
+library(datos)
+
+millas
 ```
 
-Among the variables in `mpg` are:
+Entre las variables en `millas` encontramos:
 
-1. `displ`, a car's engine size, in litres.
+1-`motor`. Tamaño del motor de un automóvil, en litros.
 
-1. `hwy`, a car's fuel efficiency on the highway, in miles per gallon (mpg). 
-  A car with a low fuel efficiency consumes more fuel than a car with a high 
-  fuel efficiency when they travel the same distance. 
+2-`autopista`. La eficiencia del uso de combustible de un automóvil en la carretera, en millas por galón. Sobre la misma distancia, un automóvil de baja eficiencia consume más combustible que un automóvil de alta eficiencia. 
 
-To learn more about `mpg`, open its help page by running `?mpg`.
+Para obtener más información sobre el conjunto de datos millas, puedes abrir la página de ayuda ejecutando `?millas`.
 
-### Creating a ggplot
+### Creando un gráfico con ggplot
 
-To plot `mpg`, run this code to put `displ` on the x-axis and `hwy` on the y-axis:
+Para graficar `millas`, corre este código usando `motor` en el eje x y `autopista` en el eje y.
 
 ```{r}
-ggplot(data = mpg) + 
-  geom_point(mapping = aes(x = displ, y = hwy))
+ggplot(data = millas) + 
+  geom_point(mapping = aes(x = motor, y = autopista))
 ```
 
-The plot shows a negative relationship between engine size (`displ`) and fuel efficiency (`hwy`). In other words, cars with big engines use more fuel. Does this confirm or refute your hypothesis about fuel efficiency and engine size?
+El gráfico muestra una relación negativa entre el tamaño del motor (`motor`) y la eficiencia del combustible (`autopista`). En otras palabras, los autos con motores grandes usan más combustible. Este resultado, ¿confirma o refuta tu hipótesis acerca de la relación entre la eficiencia del combustible y el tamaño del motor?
 
-With ggplot2, you begin a plot with the function `ggplot()`. `ggplot()` creates a coordinate system that you can add layers to. The first argument of `ggplot()` is the dataset to use in the graph. So `ggplot(data = mpg)` creates an empty graph, but it's not very interesting so I'm not going to show it here.
+Para graficar con **ggplot2**, comienzas un gráfico con la función `ggplot()`. `ggplot()` crea un sistema de coordenadas al cual puedes agregar capas. El primer argumento de `ggplot()` es el conjunto de datos para usar en el gráfico. Si corres `ggplot(data = millas)`, obtendrás un gráfico vacío. Como no es muy interesante, no vamos a mostrarlo aquí.
```

Comment:
*Sugiero: _Para comenzar a graficar en **ggplot2** se utiliza_ para _With ggplot2, you begin a plot with_

Reply:
Me parece bien esta sugerencia, aunque remplazaría *en **ggplot2*** por *con **ggplot2*** (la primera versión da la impresión que se trata de un GUI, y no de un paquete).

El párrafo quedaría:

> Para comenzar un gráfico con **ggplot2** se utiliza la función `ggplot()`. `ggplot()` crea un sistema de coordenadas al cual puedes agregar capas. El primer argumento de `ggplot()` es el conjunto de datos para usar en el gráfico. Si corres `ggplot(data = millas)`, obtendrás un gráfico vacío. Como no es muy interesante, no vamos a mostrarlo aquí.